<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QDoubleSpinBox" name="doubleSpinBox_orbitHeight">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <double>9999999.990000000223517</double>
    </property>
    <property name="value">
     <double>307.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>轨道高度</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_longitude">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.990000000000009</double>
    </property>
    <property name="value">
     <double>108.950000000000003</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_latitude">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.990000000000009</double>
    </property>
    <property name="value">
     <double>34.869999999999997</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>测站经度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>测站纬度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>计算俯仰角</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_E">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <double>90.000000000000000</double>
    </property>
    <property name="value">
     <double>3.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_run">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始计算</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_result">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>641</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>计算结果</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
